### PR TITLE
Adds a helper for \icon[foo].

### PR DESCRIPTION
**Pull request by MistakeNot4892**
This has to be done for any JS-driven UI, whether Goonchat or VueUI. Might as well do it now and get it out of the way.

### CODE REVIEW

**Review by comma — DISMISSED** [review]

**Comment by afterthought2** [comment]
You tested that this works? There are sometimes dm issues with stuff like this.

File: code/controllers/subsystems/zcopy.dm [hunk 1]
```diff
@@ -316,14 +316,14 @@ SUBSYSTEM_DEF(zcopy)
 		if (istype(A, /atom/movable/openspace/overlay))
 			var/atom/movable/openspace/overlay/OO = A
 			var/atom/movable/AA = OO.associated_atom
-			out += "<li>\icon[A] plane [A.plane], layer [A.layer], depth [OO.depth], associated Z-level [AA.z] - [OO.type] copying [AA] ([AA.type], eventually [OO.mimiced_type])</li>"
+			out += "<li>[CHAT_ICON(A)] plane [A.plane], layer [A.layer], depth [OO.depth], associated Z-level [AA.z] - [OO.type] copying [AA] ([AA.type], eventually [OO.mimiced_type])</li>"
```
Comment (Lohikar) [inline code/controllers/subsystems/zcopy.dm]: These aren't going to chat.

Comment (MistakeNot4892) [reply code/controllers/subsystems/zcopy.dm]: I can find and replace it to HTML_ICON if that would make you happy. I don't think it really matters.

Comment (Lohikar) [reply code/controllers/subsystems/zcopy.dm]: That would be fine; main thing is that its usage matches its name.

**Comment by MistakeNot4892** [comment]
image

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**